components/raftstore-v2/src/batch/store.rs

```diff
@@ -430,7 +432,29 @@ impl<EK: KvEngine, ER: RaftEngine> StoreSystem<EK, ER> {
                 .broadcast_normal(|| PeerMsg::Tick(PeerTick::PdHeartbeat));
         });
 
-        let mut workers = Workers::new(background, pd_worker);
+        let purge_worker = if raft_engine.need_manual_purge() {
+            let worker = Worker::new("purge-worker");
```

Comment:
Can it share `background` worker?

Reply:
No, it's quite heavy. One task may take 1min.